changelogs/fragments/4578-ipa_dnsrecord-add_multiple_record_support.yml

```
@@ -0,0 +1,7 @@
+---
+breaking_changes:
+ - ``record_value`` - an value with "," inside is interpreated by Ansible to split to array, so if your e.g. TXT-Record contains comma use an array with one value
+
+minor_changes:
+ - change argument type for ``record_value`` from `str` to ``list` of strings and support multiple values for one record
```

Always use the correct format for an entry:
```suggestion
 - ipa_dnsrecord - add new argument ``record_values``, mutually exclusive to ``record_value``, which supports multiple values for one record (https://github.com/ansible-collections/community.general/pull/4578).
```
Same for the next one.